src/App.jsx

```diff
@@ -6,7 +6,7 @@ import Component1Bad from './nestedComponentsBad.jsx'
 import Component1Good from "./nestedComponentsGood.jsx";
 import TitleTextAndImg from "./TitleTextImg.jsx"
 
-function App() {
+export function App() {
```

Comment:
The app is already exported (as defualt) at the bottom of the file, so you shouldnt need to add export here!

In general for the main component of a file we would use export default.